Composer/packages/lib/bot-deploy/src/botProjectDeploy.ts

```diff
@@ -89,6 +90,22 @@ export class BotProjectDeploy {
       value: scope,
     };
   }
+  private async getTenantId() {
+    if (!this.accessToken) {
+      throw new Error('Invalid Access Token');
+    }
+    try {
+      const tenantUrl = `https://management.azure.com/tenants?api-version=2020-01-01`;
+      const options = {
+        headers: { Authorization: `Bearer ${this.accessToken}` },
+      } as rp.RequestPromiseOptions;
+      const response = await rp.get(tenantUrl, options);
+      const jsonRes = JSON.parse(response);
+      return jsonRes.value[0].tenantId;
+    } catch (err) {
+      throw new Error(`Get Tenant Id Failed`);
```

Comment:
I think we should output the error, so users can give us that information if they run into this, since there are many things that could go wrong here, from network connection, to unauthorized access token, etc.